live/lib/auth_app_web/user_auth.ex

```diff
@@ -32,15 +35,22 @@ defmodule AuthAppWeb.UserAuth do
     conn
     |> renew_session()
     |> put_token_in_session(token)
-    |> maybe_write_remember_me_cookie(token, params)
+    |> maybe_write_remember_me_cookie(token, params, conn)
     |> redirect(to: user_return_to || signed_in_path(conn))
   end
 
-  defp maybe_write_remember_me_cookie(conn, token, %{"remember_me" => "true"}) do
+  defp maybe_write_remember_me_cookie(conn, token, %{"remember_me" => "true"}, _old_conn) do
+    put_resp_cookie(conn, @remember_me_cookie, token, @remember_me_options)
+  end
+
+  defp maybe_write_remember_me_cookie(conn, token, _params, %Plug.Conn{
+         req_cookies: %{@remember_me_cookie => _}
```

Comment:
So what happens if I am logged in as José and I click the magic link to sign in as Steffen? I guess we would look at the `@remember_me_cookie` from José? I wonder if there is something we could/should do?

Why did we give up on sending the remember_me from the form again?

Reply:
> I guess we would look at the @remember_me_cookie from José?

We don't look at the value of the cookie, only if it's there. So if you are signed in as José, but use the link of another user, you'd be signed in as that user with remember_me for that user as well.

> Why did we give up on sending the remember_me from the form again?

Because the LiveView does not have access to the information that remember_me is set, while the plug has. Another option would be to write that information into the session and read from there in the LV. Not sure if that's really much prettier.

Reply:
> Another option would be to write that information into the session and read from there in the LV. Not sure if that's really much prettier.

I actually like this a lot, shall we give it a try in a commit?

Reply:
I changed it to use the session, but I did not involve the LV.